website/server/libs/payments.js

```diff
@@ -128,13 +153,35 @@ api.createSubscription = async function createSubscription (data) {
     }
   }
 
-  await data.user.save();
+  if (group) {
+    await group.save();
+  } else {
+    await data.user.save();
+  }
+
   if (data.gift) await data.gift.member.save();
 };
 
 // Sets their subscription to be cancelled later
 api.cancelSubscription = async function cancelSubscription (data) {
   let plan = data.user.purchased.plan;
```

Comment:
Yep, yep. I always try to avoid using else when possible, but I am fine to change it as well.